string object_type  # 物资类别名称
int32 center_x      # 中心点X坐标
int32 center_y      # 中心点Y坐标
float32 confidence  # 置信度
int8 is_in_position # 是否到达目标位置（0: 未到达，1: 到达）